fhservices.cfg.j2

```diff
@@ -74,3 +79,15 @@ define service {
        contact_groups rhmapadmins
 }
 {% endfor %}
+
+{% for chk in ['usage','inodes'] %}
+define service {
+       service_description Storage::Pod::Disk {{ chk | capitalize() }}
+       hosts {{ rhmap_router_dns }}
+       check_command check_pod_disk_usage!{{ chk }}!80!90
```

Comment:
Is there an use case to pass in different percentages for `inodes` and `usage` modes?
Asking because if we check both block usage and inode usage in one single check, then possibly the threshold argument would be shared.

I saw in a container output like this:

```
Mounted on     Use% IUse%
/etc/hosts      19%    8% 
```

So those percentages are not necessarily in sync.